common/pkcs11x.h

```diff
@@ -181,6 +181,65 @@ typedef CK_ULONG                        CK_TRUST;
 
 #endif /* CRYPTOKI_RU_TEAM_TC26_VENDOR_DEFINED */
 
+/* Define this if you want the IBM specific symbols */
+#define CRYPTOKI_IBM_VENDOR_DEFINED 1
+#ifdef CRYPTOKI_IBM_VENDOR_DEFINED
+
+#ifndef CKK_VENDOR_DEFINED
```

Comment:
These are also defined in `common/pkcs11.h`.

Reply:
right. do you want me to remove the CKK_VENDOR_DEFINED definitions then?

Reply:
If there is no use of this file (pkcs11x.h) alone, then that would be nice.

Reply:
will check and force push if that works.

Reply:
changed